src/containers/LocationPage/LocationPage.jsx

```diff
@@ -395,6 +351,14 @@ class LocationPage extends PureComponent {
     }
   }
 
+  /**
+   * Changes the state which determines which ISP to render incidents for (if present)
```

Comment:
The first line of this docstring is a little confusing... Maybe it could say "Change the state to render incidents for a specified ISP"